resources/js/Pages/Home.vue

```diff
@@ -4,9 +4,7 @@
         <section id="intro-section">
             <h2 class="h4">{{ $i18n('about-mismatch-finder-title') }}</h2>
             <p id="about-description" >{{ $i18n('about-mismatch-finder-description') }}</p>
-        </section>
 
-        <section id="message-section">
```

Comment:
We probably want to retain this in it's own section, both for the padding added by the design, but also for semantics.